app/src/i18n/locales/en/auth.json

```
@@ -63,5 +63,7 @@
   "password-min-length-check": "At least {{length}} characters",
   "password-upper-case-check": "At least one uppercase letter",
   "password-lower-case-check": "At least one lowercase letter",
-  "password-number-check": "At least one number"
+  "password-number-check": "At least one number",
+
+  "logged-in-successful": "Success! You're logged in you CityCatalyst account"
```

Grammar - should probably be this:
```suggestion
  "logged-in-successful": "Success! You're logged into your CityCatalyst account"
```